### DISCUSSION TITLE
solved Help with proxmox lxc (ubuntu and centos ) very slow download speed after few seconds.

### DISCUSSION THREAD

**Discussion by tleung927** [haugene/docker-transmission-openvpn#2262] [Q&A]
I had successfully install docker-ce and run docker-tranmssion-vpn. But it had a very weird problem. My home internet is 300/10, and I in south cal using IP vanish,  the download are 2 -3 mb at start up but after few seconds it will drop to 200kb - 300kb max. But if I run on my desktop (windows+ docker ) it can keep around 7-10 mb. Any idea?

**Comment by pkishino** [comment]
Check using speedtest cli inside the container for comparison, if you search here there’s a description how ( can’t recall if in the wiki or not)

**Reply by tleung927** [reply]
I check 3 times download speed between 190 - 210

**Reply by pkishino** [reply]
That’s inside the container, right? Check if you have utp enabled or not. And do you mean 190-200 kb or mb? If it’s the first then your need to check proxmox networking, shouldn’t be the container.. you could try another docker container running some similar client etc to compare

**Reply by tleung927** [reply]
I test inside the container . But I solved the problem stupid me I set the cache 512 mb and I set the container 512 mb also . I guess it's out of memory . I restinall the docker set cache 128mb and increase the memory or the lxc. It's back to normal now. Thanks for your reply
